lib/node/NthRootTerm.js

```diff
@@ -0,0 +1,27 @@
+const NodeType = require('./Type');
+const TermWithCoefficient = require('./TermWithCoefficient');
+
+// For storing nth root terms, which are a subclass of TermWithCoefficient
+// where the base node is an nth root
+class NthRootTerm extends TermWithCoefficient {
+  constructor(node, onlyImplicitMultiplication=false) {
+    super(node, NthRootTerm.baseNodeFunc, onlyImplicitMultiplication);
```

Comment:
Don't think we need the `onlyImplicitMultiplication` param. Athena will handle the dots.